nibabel/gifti/tests/test_parse_gifti_fast.py

```diff
@@ -99,6 +99,45 @@
 DATA_FILE6_darr1 = np.array([9182740, 9182740, 9182740], dtype=np.float32)
 
 
+def assert_default_types(loaded):
+    default = loaded.__class__()
+    for attr in dir(default):
+        defaulttype = type(getattr(default, attr))
+        # Optional elements may have default of None
+        if defaulttype is type(None):
+            continue
+        loadedtype = type(getattr(loaded, attr))
+        assert_equal(loadedtype, defaulttype,
+                     "Type mismatch for attribute: {} ({!s} != {!s})".format(
```

Comment:
Actually, now I'm trying to debug the errors on Python 2.6, I think it's time for 2.6 to go.  It's getting really hard to set up testing on 2.6.   OK with you?